x-pack/plugin/esql/qa/testFixtures/src/main/resources/date_nanos.csv-spec

```diff
@@ -555,7 +555,8 @@ required_capability: date_nanos_bucket
 
 FROM date_nanos 
 | WHERE millis > "2020-01-01" 
-| STATS ct = count(*) BY yr = BUCKET(nanos, 1 year);
+| STATS ct = count(*) BY yr = BUCKET(nanos, 1 year)
+| SORT yr DESC;
```

Comment:
I don't think all our tests need a `SORT` at the end; IMHO that's what `ignoreOrder:true` is for. A `SORT` at the end also changes the execution plan and, because it's a pipeline breaker, can affect which computations are run on the coordinator vs. on the data nodes, which can be undesirable depending on the test.

It sure doesn't hurt here, of course, I was just unsure of the intention :)